aws/table_aws_elastic_beanstalk_environment.go

```diff
@@ -0,0 +1,298 @@
+			},
+			{
+				Name:        "description",
+				Description: "Describes this environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "endpoint_url",
+				Description: "the URL to the LoadBalancer.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsElasticBeanstalkEnvironment,
+			},
+			{
+				Name:        "abortable_operation_in_progress",
+				Description: "Indicates if there is an in-progress environment configuration update or application version deployment that you can cancel.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "environment_arn",
+				Description: "The environment's Amazon Resource Name (ARN), which can be used in other API requests that require an ARN.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "environment_links",
+				Description: "A list of links to other environments in the same group.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsElasticBeanstalkEnvironment,
+			},
+			{
+				Name:        "health",
+				Description: "The health status of the environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "health_status",
+				Description: "Returns the health status of the application running in your environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "operations_role",
+				Description: "The Amazon Resource Name (ARN) of the environment's operations role.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "platform_arn",
+				Description: "The ARN of the platform version.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "resources",
+				Description: "The description of the AWS resources used by this environment.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "solution_stack_name",
+				Description: "The name of the SolutionStack deployed with this environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "template_name",
+				Description: "The name of the configuration template used to originally launch this environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "status",
+				Description: "The current operational status of the environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "tier",
+				Description: "Describes the current tier of this environment.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "version_label",
+				Description: "The application version deployed in this environment.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags assigned to the Repository",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getElasticBeanstalkEnvironmentTags,
+				Transform:   transform.FromField("ResourceTags"),
+			},
+
```

Comment:
please arrange the columns according to the standard column structure

Reply:
Ordering of columns is still not correct.